gtsfm/utils/prepare_nerfstudio.py

```diff
@@ -0,0 +1,159 @@
+"""Functions to create transforms.json file and resized images that are input to Nerfstudio
+
+Here I have modified code taken from Nerfstudio for parsing data in the Nerfstudio format.
+Original file at:
+https://github.com/nerfstudio-project/nerfstudio/blob/main/scripts/process_data.py.
+
+Author: Jon Womack
+"""
+import argparse
+import os
+import json
+import numpy as np
+from enum import Enum
+from pathlib import Path
+
+
+from gtsfm.utils import images, io
+import thirdparty.nerfstudio.colmap_utils as colmap_utils
```

Comment:
We follow a specific ordering of imports:

```
<included in python by default>

<third party imports (e.g. numpy)>

<imports from the same package>
```

Can you follow this structure?

Reply:
Yes, the imports are now reordered